docs/cli.md

```diff
@@ -1101,20 +1104,23 @@ viam machines part restart --machine=<machine id> --part=<part id>
 Examples:
 
 ```sh {class="command-line" data-prompt="$"}
-
 # list all machines you have access to
 viam machines list
 
 # get machine status
 viam machines status  --machine 82c608a-1be9-46a5-968d-bad3a8a6daa --organization "Robot's Org" --location myLoc
 
-# stream error level logs from a machine part
+# stream logs from a machine
 viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
---organization "Robot's Org" --location myLoc --part "myrover-main" --tail true
+--organization "Robot's Org" --location myLoc
+
+# stream logs from a machine part
+viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
+--organization "Robot's Org" --location myLoc --part myrover-main --tail true
```

Comment:
the part must be specified as an id as far as I can tell testing this.

Reply:
I found this to work: 
``` viam machines part run --organization=84318326-eb77-450a-88ec-32a043b793a8 --location=0b2qnylnp0 --machine=collect-data --part=collect-data-main --data '{"name": "vision-1", "camera_name": "camera-2", "detector_name": "mlmodel-1", "n":1}' \
viam.service.vision.v1.VisionService.GetDetectionsFromCamera
```
so I don't think it actually needs to be specd to id!